classwork/fork-folder/01_git/result.txt

```diff
@@ -0,0 +1 @@
+https://github.com/kononovich11/summer
```

Comment:
@kononovich11 that's the wrong link. Please provide a link to the repo you made in the solid train exercise